.env.sample

```diff
@@ -1,22 +1,3 @@
-CORE_PORT=8000
-
-NODE_ENV=development
-
-CORS_ORIGIN_WHITELIST=http://localhost:3000,http://127.0.0.1:3000
-AUTH=false
-
-RATE_LIMIT_ENABLED=false
-RATE_LIMIT_WINDOW=1000
-RATE_LIMIT_MAX=10
-
-DB=foobar.db
-
-
-ACCESS_TOKEN_EXPIRATION_TIME=10H
-REFRESH_TOKEN_EXPIRATION_TIME=2D
-
-INITIAL_USER_USERNAME 
-INITIAL_USER_PASSWORD
-TOKEN_SECRET
-
-START_WITH_STUDIO=false
+DB_FILE=/data/master.db
```

Comment:
hello @ivishalgandhi - I suspect these should be present as additional env variables, rather than replacing the previous ones.
It would also be "neat" to include a leading comment to explain that they are supporting docker configuration.
Aah, one last thing.  How about preceding their names with `DOCKER_` - then it's clear what they relate to (otherwise there is potential for confusion between `DB` and `DB_FILE`).

Reply:
I agree with @IanMayo here. I don't think that we should use other names for the environment variable that already exists. Also, you removed the content of the original .env.sample file which we need to revert it.
If there's a need for a separate env file please elaborate on it and we can create a new env file e.g. `.env.docker.sample`.

Reply:
I am working on rewrite of the files. will share soon. Appreciate all the inputs.